web-marketplace/src/pages/BuyCredits/BuyCredits.utils.tsx

```diff
@@ -413,3 +422,84 @@ export const getCryptoCurrencyIconSrc = (
       'https://regen-registry.s3.us-east-1.amazonaws.com/assets/icons/evmos.png';
   return href;
 };
+
+export const getAmountLabel = (retiring: boolean) => {
+  return retiring
+    ? i18n._(AMOUNT_LABELS.RETIRED)
+    : i18n._(AMOUNT_LABELS.TRADABLE);
+};
+
+/**
+ * Sends an purchase confirmation email after a successful credit purchase.
```

Comment:
```suggestion
 * Sends a purchase confirmation email after a successful credit purchase.
```
we could also specify that it's for crypto purchases only since for fiat, it's handled on the server side